dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/AbstractEventJdbcTableManager.java

```diff
@@ -149,6 +150,12 @@ public String validState()
         return null;
     }
 
+    @Override
+    protected boolean hasUpdatedLatestData( Date startDate, Date endDate )
+    {
+        throw new IllegalStateException( "Should not be invoked for event table managers" );
```

Comment:
`AnalyticsTableType` has EVENT and ENROLLMENT with the property `latestPartition` set to `true`. Maybe I misunderstood?

Reply:
Good point, in fact ENROLLMENT should have latestPartition = false. Updated that now.

Reply:
Agree on returning false vs exception too. Changed it to false.